gpg-interface.h

```diff
@@ -3,6 +3,7 @@
 
 #define GPG_VERIFY_VERBOSE	1
 #define GPG_VERIFY_RAW		2
+#define GPG_VERIFY_QUIET	4
 
```

Comment:
I replied via email, but yeah. We would have to change the commit message a little bit.